openmetadata-spec/src/main/resources/json/schema/entity/data/glossaryTerm.json

```diff
@@ -153,6 +153,10 @@
     },
     "votes" : {
       "$ref": "../../type/votes.json"
+    },
+    "childrenCount": {
```

Comment:
we may need to do a migration for it? did we test what happens if there is no childrenCount is available for other APIs which are fetching glossaryTerms. 
Does this need to update search indexes as well?

Reply:
we will not need migration for it, as we are not storing it in db, creating on fly,  ingestion will require change.
Have run all the tests on GlossaryResourceTest, tests have passed.